subiquity/server/controllers/install.py

```diff
@@ -187,10 +189,15 @@ def acquire_initial_config(self,
             }
         }
 
-    def acquire_filesystem_config(self, step: CurtinPartitioningStep,
-                                  ) -> Dict[str, Any]:
+    def acquire_filesystem_config(
+            self, step: CurtinPartitioningStep,
+            mode: ActionRenderMode = ActionRenderMode.DEFAULT,
+            ) -> Dict[str, Any]:
         cfg = self.acquire_initial_config(step)
-        cfg.update(self.model.filesystem.render())
+        version = None
+        if mode != ActionRenderMode.DEFAULT:
+            version = 2
```

Comment:
Isn't this too late?  I worry about earlier parts of the code having made different decisions (the gaps code for example having a completely different code path) than it would have if it was v2.

Reply:
I think probably the way the code is now it isn't actually a problem but you're definitely right that this is a surprise waiting to happen. I'll try something else :)

(v2 by default everywhere when? :) )